pom.xml

```diff
@@ -602,14 +602,21 @@ under the License.
 				<!-- mockito/powermock mismatch -->
 				<groupId>net.bytebuddy</groupId>
 				<artifactId>byte-buddy</artifactId>
-				<version>1.10.14</version>
+				<version>1.14.4</version>
 			</dependency>
 
 			<dependency>
 				<!-- mockito/powermock mismatch -->
 				<groupId>net.bytebuddy</groupId>
 				<artifactId>byte-buddy-agent</artifactId>
-				<version>1.10.14</version>
+				<version>1.14.4</version>
+			</dependency>
+
+			<!-- For dependency convergence -->
+			<dependency>
+				<groupId>net.java.dev.jna</groupId>
+				<artifactId>jna</artifactId>
+				<version>5.12.1</version>
```

Comment:
From previous experience, we needed the version of the JNA to be in sync with the one used by Testcontainer, else all testcontainer upgrades fail with a rather indeterministic error